output/opendap_post2.sh

```diff
@@ -142,13 +143,13 @@ for _file in ${_FILES[*]}; do
     continue
   fi
   # readlink will resolve $_file to a full path
-  _file=$(readlink -f $_file)
+  _file=$(readlink -f $SCENARIODIR/$_file)
```

Comment:
this will break the ability to use both relative and full paths

Reply:
Yes this change didn't feel quite right to me either. The status files are an interesting case because they come from a different "faux scenario" directory, and they end up getting posted to a different path on the remote server than the normal scenario based output files. Let's discuss on our weekly call tomorrow.

Reply:
I have fixed this as we discussed last week